src/util/SystemCommand.ts

```diff
@@ -101,11 +103,11 @@ export class SystemCommand {
     /*
      * Execution implementation with VS Code's embedded terminal
      */
-    executeWithTerminal(): Promise<any> {
+    executeWithTerminal(preserveFocus?:boolean): Promise<any> {
```

Comment:
**Callout:** Added optional argument to preserveFocus to allow focus from Command Palette to terminal. Without this the user will not be able to interact with the terminal panel.